a10_octavia/controller/worker/tasks/a10_database_tasks.py

```diff
@@ -87,7 +88,8 @@ def execute(self, amphora, loadbalancer, role, status):
             last_udp_update=datetime.utcnow(),
             status=status,
             created_at=datetime.utcnow(),
-            updated_at=datetime.utcnow())
+            updated_at=datetime.utcnow(),
+            partition=partition)
```

Comment:
Shouldn't need this as we aren't support this for vThunders at this time.

Reply:
removing from here.